oedisi/tools/cli_tools.py

```diff
@@ -94,6 +291,33 @@ def run(runner):
     """
     subprocess.run(["helics", "run", f"--path={runner}"])
 
+@cli.command()
+@click.option("--runner", default="build/system_runner.json", type=click.Path(),
+              help="Location of helics run json. Usually build/system_runner.json")
+def run_api(runner):
+
+    broker_port = None
+    broker_ip = None
+    
+    with open(runner, 'r') as f:
+        config = json.load(f)
+        for i, fed_info in  enumerate(config['federates']):
+            if fed_info["name"] == "broker":
+                broker_port = fed_info["port"]
+                broker_ip = fed_info["hostname"]
+        
+        if not broker_port:
+            raise Exception("Broker port not found in the config file")
+        
+        
+        for i, fed_info in  enumerate(config['federates']):  
+            if fed_info["name"] != "broker":
+                url = f'https://{fed_info["hostname"]}:{fed_info["port"]}'
+                #myobj = {'somekey': 'somevalue'}
+                x = requests.get(url)
+                print(x.text)
+                #x = requests.post(url, json = myobj)
+
```

Comment:
Remove these extra comments.

From context, it seems like this checks that the broker port exists and that we can query all the other federates, right? How about test_api?

Reply:
It also seems like this can't query the api_port of the broker because the broker has this broker config instead?

Reply:
removed.

Do you want to test if the services are live?

Reply:
change name to test federates @AadilLatif

Reply:
Looks like you just deleted this outright?